app/views/_nav_bar.html.erb

```diff
@@ -2,19 +2,26 @@
   <%= link_to "home", root_path  %>
   <%= link_to "browse items", items_path  %>
   <%= link_to "browse merchants", merchants_path  %>
-  <%= link_to "shopping cart", cart_path  %>
 
-  <% if current_user && current_user.role == "default" %>
+  <% unless current_user && current_user.role == "admin" %>
+    <%= link_to "shopping cart", cart_path  %>
```

Comment:
You can include this into the conditional below in an `else` statement

Reply:
changed